golang/lib/execution/lambda_service_server.go

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020 - present Kurtosis Technologies LLC.
+ * All Rights Reserved.
+ */
+
+package execution
+
+import (
+	"context"
+	"github.com/kurtosis-tech/kurtosis-client/golang/lib/networks"
+	"github.com/kurtosis-tech/kurtosis-lambda-api-lib/golang/kurtosis_lambda_rpc_api_bindings"
+	"github.com/kurtosis-tech/kurtosis-lambda-api-lib/golang/lib/lambda"
+	"github.com/palantir/stacktrace"
+	"google.golang.org/protobuf/types/known/emptypb"
+	"reflect"
+)
+
+type LambdaServiceServer struct {
+	// This embedding is required by gRPC
+	kurtosis_lambda_rpc_api_bindings.UnimplementedLambdaServiceServer
+	lambda     lambda.Lambda
+	networkCtx *networks.NetworkContext
+}
+
+func NewLambdaServiceServer(lambda lambda.Lambda, networkCtx *networks.NetworkContext) *LambdaServiceServer {
+	return &LambdaServiceServer{lambda: lambda, networkCtx: networkCtx}
+}
+
+func (lambdaService *LambdaServiceServer) IsAvailable(ctx context.Context, empty *emptypb.Empty) (*emptypb.Empty, error) {
```

Comment:
For now, for simplicity, let's just return `true`, but I totally see the argument of providing this in the near future

Reply:
Yes, I totally agree, I was leaving this to the end.
Do you already have an idea of ​​how we should check this? is there a way to do it in GRPC? I mean, to avoid implement an HTTP server and adding an endpoint.